README.md

```diff
@@ -1,13 +1,55 @@
+
+### Checking Certificate Parameters
+In order to check the mobile certificate parameters, the security library needs the TLS issuer and certificate password used to generate the certificate. Currently the the `SDLTLSIssuer` and `SDLTLSCertPassword` properties used to check the certificate are hardcoded in the **SDLPrivateSecurityConstants.m** file. 
+
+### Certificate Storage
+Once the certificate is downloaded it is stored on disk so the certificate can persist between app sessions. The `VendorName` property in the  **SDLSecurityManager.m** file is used to create a directory where your certificate is stored. We recommend updating this property to your company name. 
+
+### Renaming the Library
+In order to use this security library with SDL, an OEM must rename this library and classes. This is because a developer who wants to support multiple OEMS will have to add a security manager from each OEM. If OEMS use the same class names then it will be impossible for the developer to include more than one security library in their application.
+
+## Generating the Static Security Library
+Once the security library has been configured by the OEM, it is used to generate a static library that developers add to their SDL iOS apps. The static library, **libSDLSecurityStatic.a**, can easily be built in Xcode using the **SDLSecurityStatic** target:
+
+1. In the scheme menu of Xcode, set the active scheme to **SDLSecurityStatic**.
+1. Build and run the **SDLSecurityStatic** scheme (**Product > Build For > Running**). 
+1. In the project navigator you will find the **libSDLSecurityStatic.a** build under **SDLSecurity > Products**.
+1. Right click on the **libSDLSecurityStatic.a** build and select **Show in Finder**. This will take you to the derived data of the project which has the three builds listed below:
+    * *Debug-iphoneos* - will only work on an iPhone 
+    * *Debug-iphonesimulator* - will only work on a simulator
+    * *Debug-universal*  - will work on both the iPhone and simulator
+1. There are also two header files in the **include** folder, **SDLSecurityConstants.h** and **SDLSecurityManager.h** that must be included along with **libSDLSecurityStatic.a** archive build.
+    
+## Adding the Static Security Library to a SDL App
+In order to use the static security library in an SDL app you must have three files: **libSDLSecurityStatic.a**, **SDLSecurityConstants.h** and **SDLSecurityManager.h**
+
+1. In Xcode, drag and drop the following 3 files into your project: **libSDLSecurityStatic.a**, **SDLSecurityConstants.h** and **SDLSecurityManager.h**. Make sure **libSDLSecurityStatic.a** has been added to the project's target membership.  
+1. Import **SDLSecurityManager.h** into the file where the the `SDLConfiguration`'s `SDLEncryptionConfiguration` is being set. If you have a Swift project, this will require adding a bridging header to the project.
```

Comment:
```suggestion
1. Import **SDLSecurityManager.h** into the file where the the `SDLConfiguration`'s `SDLEncryptionConfiguration` or `SDLStreamingMediaConfiguration` is being set. If you have a Swift project, this will require adding a bridging header to the project.
```